packages/zowe-explorer/__tests__/__integration__/extension.integration.test.ts

```diff
@@ -152,18 +156,28 @@ describe("Extension Integration Tests", () => {
         }).timeout(TIMEOUT);
 
         it("should display an error message when creating a data set that already exists", async () => {
-            // Mock user selecting first option from list
-            const quickPickStub = sandbox.stub(vscode.window, "showQuickPick");
-            quickPickStub.returns("Data Set Sequential");
-
+            // 1st step: User names DS
             const testFileName = pattern + ".EXT.CREATE.DATASET.TEST";
             const inputBoxStub = sandbox.stub(vscode.window, "showInputBox");
             inputBoxStub.returns(testFileName);
 
+            // 2nd step: User selects DS type
+            const quickPickStub = sandbox.stub(vscode.window, "showQuickPick");
+            quickPickStub.onCall(0).returns("Data Set Sequential");
```

Comment:
If possible can you change this to a foreach statement? The last PR that I opened has magic numbers error. I can see that yours doesn't have one but it would be great if we could prevent it in the future. I did it like this:

```
const returnVals = ["testUser", "testPass", "basePath", "encoding", "timeOut"];
        returnVals.forEach((value, index) => {
            showInputStub.onCall(index).returns(value);
        });
```